electron-app/src/components/dashboard/BlockchainBalanceCardList.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <v-list-item
+    v-if="amount !== zero"
+    :id="`${name}_box`"
+    :ripple="false"
+    class="blockchain-balance-box__item"
+    @click="doNothing"
+  >
+    <v-list-item-avatar
+      tile
+      class="blockchain-balance-box__icon blockchain-box__icon"
```

Comment:
is the second class defined anywhere? I cannot see it in this components scope

Reply:
No it's redundant / left-over. Will fix!